shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -37,11 +39,9 @@ void PlatformChannel::HandleMethodCall(
   } else if (method == "HapticFeedback.vibrate") {
     result->NotImplemented();
   } else if (method == "Clipboard.getData") {
-    result->NotImplemented();
+    Clipboard::GetData(call, std::move(result));
```

Comment:
Minor suggestion: Why don't you pass the arguments only? I don't know if this will work or not but:

```cpp
const auto arguments = call.arguments();

...
} else if (method == "Clipboard.getData") {
  Clipboard::GetData(arguments, std::move(result));
} else if (method == "Clipboard.setData") {
  Clipboard::SetData(arguments, std::move(result));
}
...

std::string format = arguments->GetString();
if (format != kTextPlainFormat) {
...
```

Reply:
as 'call' is passed by reference there will be no significant difference. If you prefer this way, I will apply your suggestion.

Reply:
It's not about efficiency but just semantics of the paramter (`arguments` has smaller scope of information). There's still not much difference and you can do as you prefer.